flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -499,10 +496,6 @@ public static class RocksdbCompactFilterCleanupStrategy
          */
         static final Duration DEFAULT_PERIODIC_COMPACTION_TIME = Duration.ofDays(30);
 
-        static final RocksdbCompactFilterCleanupStrategy
```

Comment:
Ah... I just noticed that the class `RocksdbCompactFilterCleanupStrategy` is `@PublicEnvolving`, meaning that we can not directly remove this static member. Please annotate it as `@Deprecated` and leaving some java doc mentioning the newly introduced options.

This applies for `DEFAULT_PERIODIC_COMPACTION_TIME` as well.

Reply:
Thank you for taking the time to help review the code. It has been fixed now.